ui/packages/catalog-ui-search/src/main/webapp/component/sort/sort.hbs

```diff
@@ -15,8 +15,9 @@
 <div class="sorts">
 
 </div>
+<div class="sort-add-button">
 <button class="sort-add is-primary">
     <span class="fa fa-plus">
-
     </span>
-</button>
\ No newline at end of file
+</button>
```

Comment:
Make sure this is formatted

Reply:
Weird that the formatter doesn't take care of this. I can do it manually though